src/assets/sass/views/_resources.scss

```diff
@@ -0,0 +1,19 @@
+.resource-link {
+  margin: 20px 0;
+  line-height: 1.6;
+
+  h3 {
+    font-size: 24px;
+  }
+
+  .resource-author {
+    font-size: 18px;
```

Comment:
I prefer if we can avoid hard-coding px values for font-sizes here (24px and 18px) and use the default Bulma classes which are based on rem units. This leaves some options open in the future if we want to make these font-sizes responsive.